CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(libNEGF VERSION 0.1 LANGUAGES Fortran C)
+
+set(FYPP "fypp" CACHE STRING "Fypp preprocessor")
+
+option(WITH_MPI "Whether MPI-parallelised library should be built" FALSE)
+
+# Note, this option does not work yet, so leave it to false
+option(WITH_INELASTIC "Whether to build with inelastic scattering" FALSE)
+
+set(MPIFX_INCLUDE_DIR "/opt/mpifx/include" CACHE PATH
```

Comment:
Can't this also go in the WITH_MPI block?

Reply:
:+1: